Alittle modification done to the readme.md file (#3089)
openMF/community-app: README.md

   ```npm``` installed - goto http://nodejs.org/download/ to download installer for your OS.    
   ```ruby``` installed - goto https://www.ruby-lang.org/en/documentation/installation/ to download latest version of ruby.

> <br/> Note: On Ubuntu Linux you can use 'sudo apt-get install npm nodejs-legacy' (nodejs-legacy is required to avoid the "/usr/bin/env: node: No such file or directory" problem).

Reviewer: ```suggestion
Note: On Ubuntu Linux you can use `sudo apt-get install npm nodejs-legacy` (nodejs-legacy is required to avoid the "/usr/bin/env: node: No such file or directory" problem).
```

<br/> Tip: If you are using Ubuntu/Linux, then doing ```npm config set prefix ~``` prevents you from having to run npm as root.

1. Clone this repository to your local filesystem (default branch is 'develop')

> 1. ####To download the dependencies, and be able to build,

Reviewer: ```suggestion
1. To download the dependencies, and be able to build,
```

   - first install bower & grunt
   ```
    npm install -g bower
    npm install -g grunt-cli
   ```

>    -  Next pull the runtime and build time dependencies by running bower, npm and gem bundler install commands on the   project root folder

Reviewer: ```suggestion
1.  Next pull the runtime and build time dependencies by running bower, npm and gem bundler install commands on the   project root folder
```
